release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -33,11 +33,11 @@ application for {product-title} allows you to deploy OpenShift clusters to
 either on-premise or cloud environments.
 
 {product-title} {product-version} is supported on Red Hat Enterprise Linux 7.6 and
-later, as well as Red Hat Enterprise Linux CoreOS 4.1.
+later, as well as Red Hat Enterprise Linux CoreOS 4.3.
 
 You must use {op-system-first} for the control plane, or master, machines and
-can use either {op-system} or Red Hat Enterprise Linux 7.6 for compute, or worker,
-machines.
+can use either {op-system} or Red Hat Enterprise Linux 7.6 and later for
```

Comment:
"use 7.6 and later" sounds odd, consider "7.6 or later"

Reply:
@bobfuru I agree "or later" sounds better. In the past, I've used the `+` sign instead: "use 7.6+". I stuck with "and later" since it was used in previous release notes. What are your thoughts on using `+`? Not sure how much that's used in OCP docs.

Reply:
I'll go with `or later` for now. I've seen that used several times throughout the docs. Thanks!